docs/introduction/QuickStart.mdx

```diff
@@ -0,0 +1,337 @@
+---
+id: quick-start
+title: Quick Start
+sidebar_label: Quick Start
+hide_title: true
+---
+
+import Tabs from '@theme/Tabs'
+import TabItem from '@theme/TabItem'
+
+import { DetailedExplanation } from '../components/DetailedExplanation'
+import { Note } from '../components/Note'
+
+# Quick Start
+
+<Note>
+
+### What You'll Learn
+
+- What Redux is, and why you should use it
+- How Redux relates to React
+- How to set up and start using Redux quickly
+- What a small working React+Redux app looks like
+
+### Prerequisites
+
+There are several things you should know in advance before learning Redux:
+
+- Basic familiarity with HTML & CSS, and the DOM.
+- Basic knowledge of JavaScript and programming.
+- Familiarity with ES6 syntax and features.
+- Basic knowledge of React, including components, props, and state.
+- Node.js and `npm` installed globally.
```

Comment:
should we link to external resources here? In case the end user doesn't know these pre-reqs.

Reply:
Yep, I'm just trying to get _a_ list in place for now.